tests/Doctrine/Tests/ORM/UnitOfWorkTest.php

```diff
@@ -526,8 +526,8 @@ public function testObjectHashesOfMergedEntitiesAreNotUsedInOriginalEntityDataMa
         $user->name = 'ocramius';
         $mergedUser = $this->_unitOfWork->merge($user);
 
-        self::assertSame([], $this->_unitOfWork->getOriginalEntityData($user), 'No original data was stored');
-        self::assertSame([], $this->_unitOfWork->getOriginalEntityData($mergedUser), 'No original data was stored');
+        self::assertEmpty($this->_unitOfWork->getOriginalEntityData($user), 'No original data was stored');
+        self::assertEmpty($this->_unitOfWork->getOriginalEntityData($mergedUser), 'No original data was stored');
```

Comment:
We cannot do this yet since `UnitOfWork#getOriginalEntityData()` has no return type declaration, so we must ensure that we have the correct type.